core/jobs/batch_jobs/exp_migration_jobs.py

```diff
@@ -340,189 +261,111 @@ def run(self) -> beam.PCollection[job_run_result.JobRunResult]:
         )
 
 
-# TODO(#15927): This job needs to be kept in sync with MigrateExplorationJob and
-# later we will unify these jobs together.
-class AuditExplorationMigrationJob(base_jobs.JobBase):
+# TODO(#15927): This job needs to be kept in sync with
+# AuditExplorationMigrationJob and later we will unify these jobs together.
```

Comment:
I think this comment is no longer needed.

Reply:
Done